modules/mobile/app/models/mobile/v0/adapters/immunizations.rb

```diff
@@ -4,22 +4,24 @@ module Mobile
   module V0
     module Adapters
       class Immunizations
+        # rubocop: disable Metrics/MethodLength
```

Comment:
Don't see a way to break this up intuitively so I disabled the method length rule